### PR TITLE
feat: disable stake button if utxo not available

### PR DESCRIPTION
**Pull request by jrwbabylonlab**
image

### CODE REVIEW

**Review by totraev — CHANGES_REQUESTED** [review]
Will update later today

**Review by gbarkhatov — APPROVED** [review]

File: src/app/state/index.tsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ const STATE_LIST = [DelegationState, DelegationV2State];
 
 export interface AppState {
   availableUTXOs?: UTXO[];
+  allUTXOs?: UTXO[];
```
Comment (totraev) [inline src/app/state/index.tsx:24]: @jrwbabylonlab it's not clear to me why do you use `allUTXOs` instead of `availableUTXOs`?

**Review by totraev — CHANGES_REQUESTED** [review]

**Comment by totraev** [comment]
> What need to be updated?

Don't write any domain specific logic inside render function directly. Use my fixes as a reference for future PRs

**Comment by totraev** [comment]
@jrwbabylonlab can you pls check my changes? I didn't test them

**Comment by totraev** [comment]
@jrwbabylonlab I see another problem: everytime we submit a staking tx we need invalidate utxos' cache and refetch them

File: src/app/state/index.tsx (hunk 1 above)
Comment (totraev) [reply src/app/state/index.tsx]: @jrwbabylonlab This is expected behaviour IMO. If user decided to filter out ordinals (to save them) we should not let him to stake his previously created delegation with ordinals

**Review by totraev — APPROVED** [review]

**Comment by totraev** [comment]
@jrwbabylonlab approved, but pls test before merge

**Pull request merged**